src/audio/drc/drc_math_generic.c

```diff
@@ -177,7 +177,7 @@ inline int32_t drc_asin_fixed(int32_t x)
 }
 
 /*
- * Input x is Q6.26: (-32.0, 32.0)
+ * Input x is Q6.26; valid range: [0.0, 32.0); negative values are not supported
```

Comment:
This does not the condition in the previous patch, I assume here we support >=0 ?